app/features/text_rewriter/.gitignore

```diff
@@ -0,0 +1,48 @@
+# Environment files
+.env
+text_rewriter.env
+
+# System files
+.DS_Store
+Thumbs.db
+
+# Python-related files
+__pycache__/
+*.py[cod]
+*.pyo
+*.pyd
+
+# Virtual environment
+env/
+
+# Logs
+*.log
+*.sqlite3
+
+# Compiled files
+*.pyc
+*.pyo
+*.pyd
+
+# Ignore virtual environment
+Ai/
+env/
+Lab/
+
+# VS Code configuration
+.vscode/
+
+# Output files
+output.txt
+
+# Ignore additional Ai/ and output.txt files
+Ai/
+output.txt
+.vscode/
+.env
+
+# Ignore the Google Cloud service account JSON file
+gen-lang-client-0717318025-ef6f992883f8.json
+
+# Ignore the Ai directory files
+Ai/
```

Comment:
I used this because I had trouble using langchain and could only get the model to work via google gemini and a vertex ai API key. With the examples you provided I'm rewriting the code to get rid of the vertext ai API.